vmdb/app/controllers/api_controller/accounts.rb

```diff
@@ -4,7 +4,7 @@ module Accounts
     # Accounts Subcollection Supporting Methods
     #
     def accounts_query_resource(object)
-      object.send("accounts")
+      object.accounts
```

Comment:
@jrafanie No, I did not review that each and every send is public.  The tests did find a few that I corrected.  However, I think this was caused by a copy/paste of a bad pattern.